lib/audit/moab_to_catalog.rb

```diff
@@ -20,6 +21,12 @@ def self.check_existence_for_druid(druid)
     logger.info "#{Time.now.utc.iso8601} M2C check_existence_for_druid ended for #{druid}"
   end
 
+  def self.check_existence_for_druid_list(druid_list_file_path)
+    CSV.foreach(druid_list_file_path) do |row|
+      MoabToCatalog.check_existence_for_druid(row.first)
```

Comment:
output the errors?

Reply:
How is this CSV input?  Seems like a single column of text.

Reply:
I could use`File.open` then use `.each_line` on that file but this seemed easier?

Reply:
If that's what he starts from, I guess... 🤷‍♂️ Would be nice to enable taking the values from STDIN, but whatever works.